src/data/faq_de.json

```
@@ -243,9 +243,11 @@
                         "anchor": "cert_recycle_bin",
                         "active": true,
                         "textblock": [
-                            "Ab Version 2.13 werden Impf-, Genesenen- oder Testzertifikate, die Sie gelöscht haben, vorläufig in einen Papierkorb verschoben. Dies ermöglicht Ihnen im Fall eines versehentlichen Entfernens das Zertifikat mit Leichtigkeit wiederherzustellen und es an seinen ursprünglichen Ort zurück zu verschieben. Sollte ein Zertifikat 30 Tage im Papierkorb verweilen wird es automatisch endgültig gelöscht, sofern man es vorher nicht manuell im Papierkorb endgültig gelöscht hat. Ein Zertifikat, welches in der App gelöscht worden ist, lässt sich trotzdem neu einscannen. Sollte es sich noch im Papierkorb befinden und daher nicht endgültig entfernt worden sein erhält man einen Hinweis, dass es wiederhergestellt wurde.",
-                            "<b>iOS:</b><ul><li>Wischen Sie unter Status nach unten und wählen Sie unter 'Mehr' den 'Papierkorb' aus.</li><li>Sollten sich Zertifikate in Ihrem Papierkorb befinden, können Sie ein Zertifikat antippen, um es wiederherzustellen.</li><li>Um ein Zertifikat endgültig aus dem Papierkorb zu entfernen tippen Sie zunächst oben rechts auf 'Bearbeiten'. Anschließend tippen Sie auf das rote Minuszeichen neben dem jeweiligen Zertifikat zu tippen. Alternativ können Sie direkt in der Auflistung, ohne vorher auf 'Bearbeiten' zu tippen, während des Gedrückthaltens auf einem Zertifikat nach links wischen. Tippen Sie anschließend auf das rote Feld 'Löschen'.</ul>",
-                            "<b>Android:</b><ul><li>Tippen Sie unter Status auf das Drei-Punkte-Menü oben rechts in der Ecke.</li><li>Tippen Sie daraufhin auf 'Papierkorb'.</li><li>Tippen Sie innerhalb eines der dort aufgelisteten Zertifikate auf das Drei-Punkte-Menü oben rechts.</li><li>Wählen Sie aus, ob Sie das ausgewählte Zertifikat wiederherstellen oder endgültig aus dem Papierkorb entfernen wollen.</li></ul>",
+                            "Ab Version 2.13 werden <b>Impf-, Genesenen- oder Testzertifikate</b>, die Sie gelöscht haben, vorläufig in einen Papierkorb verschoben. Dies ermöglicht Ihnen im Fall eines versehentlichen Entfernens das Zertifikat mit Leichtigkeit wiederherzustellen und es an seinen ursprünglichen Ort zurück zu verschieben. Sollte ein Zertifikat 30 Tage im Papierkorb verweilen wird es automatisch endgültig gelöscht, sofern man es vorher nicht manuell im Papierkorb endgültig gelöscht hat. Ein Zertifikat, welches in der App gelöscht worden ist, lässt sich trotzdem neu einscannen. Sollte es sich noch im Papierkorb befinden und daher nicht endgültig entfernt worden sein erhält man einen Hinweis, dass es wiederhergestellt wurde.",

```

```suggestion
                            "Ab Version 2.13 werden <b>Impf-, Genesenen- oder Testzertifikate</b>, die Sie gelöscht haben, vorläufig in einen Papierkorb verschoben. Dies ermöglicht Ihnen im Fall eines versehentlichen Entfernens das Zertifikat mit Leichtigkeit wiederherzustellen und es an seinen ursprünglichen Ort zurück zu verschieben. Sollte ein Zertifikat 30 Tage im Papierkorb verweilen wird es automatisch endgültig gelöscht, sofern man es vorher nicht manuell im Papierkorb endgültig gelöscht hat. Ein Zertifikat, welches in der App gelöscht worden ist, lässt sich trotzdem neu einscannen. Sollte es sich noch im Papierkorb befinden und daher nicht endgültig entfernt worden sein, erhält man einen Hinweis, dass es wiederhergestellt wurde.",
```

missing comma between "sein" and "erhält"